app/src/main/java/de/danoeh/antennapod/dialog/SleepTimerDialog.java

```diff
@@ -153,4 +155,9 @@ private void updateTime() {
         timeDisplay.setVisibility(controller.sleepTimerActive() ? View.VISIBLE : View.GONE);
         time.setText(Converter.getDurationStringLong((int) controller.getSleepTimerTimeLeft()));
     }
+
+    private void closeKeyboard(View content) {
+        InputMethodManager imm = (InputMethodManager) getContext().getSystemService(Activity.INPUT_METHOD_SERVICE);
+        imm.hideSoftInputFromWindow(content.getWindowToken(), 0);
```

Comment:
Is this still needed even when the manifest is changed?

Reply:
Yes, because if you start the sleep timer, the dialog is not dismissed, which means we have to manual hide the keyboard.